tasks/request_status_for_granule/request_status_for_granule.py

```diff
@@ -182,8 +182,8 @@ def get_job_entry_for_granule_sql() -> text:
                 SELECT
                     granule_id as "{OUTPUT_GRANULE_ID_KEY}",
                     job_id as "{OUTPUT_JOB_ID_KEY}",
-                    EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC') * 1000 as "{OUTPUT_REQUEST_TIME_KEY}",
-                    EXTRACT(EPOCH FROM date_trunc('milliseconds', completion_time) AT TIME ZONE 'UTC') * 1000 as "{OUTPUT_COMPLETION_TIME_KEY}"
+                    EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC')::bigint * 1000 as "{OUTPUT_REQUEST_TIME_KEY}",
```

Comment:
You will want to do the cast after taking it times 1000 to avoid rounding errors so

`(:start_timestamp is null or EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC')::bigint * 1000` will become `(:start_timestamp is null or (EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC') * 1000)::bigint`

The reason being, `EXTRACT(EPOCH FROM date_trunc('milliseconds', request_time) AT TIME ZONE 'UTC')` may return something like `1638997146.5436` if you cast before taking it `*1000` the casting will round this to `1638997147` which will result in a value of `1638997147000` instead of a value of `16389971465436` which is what you want. This is done on lines 185 and 186.

Reply:
Got complacent with expected precision. Good catch. Fixed.